GitWorkflow/GitWorkflow/main.cpp

```diff
@@ -12,13 +12,15 @@ using Time = std::chrono::time_point<Clock>;
 int main()
 {
 	sf::RenderWindow window(sf::VideoMode(WINDOW_WIDTH, WINDOW_HEIGHT), "SFML works!");
+	const float windowScale = (WINDOW_WIDTH / WINDOW_HEIGHT) / (1080 / 720);
 	const sf::Vector2f platformSize = { 100.0f, 10.0f };
 	Level level;
 
 	Time t1 = Clock::now();
 
 	level.create<EndPoint>(sf::Vector2f(20.0f, 20.0f), sf::Vector2f(300.0f, 100.0f));
 	level.create<Player>(sf::Vector2f(20.0f, 20.0f), sf::Vector2f(100.0f, 100.0f));
+	level.create<Platform>(sf::Vector2f(WINDOW_WIDTH, 10.0f), sf::Vector2f(0.0f, WINDOW_HEIGHT));
```

Comment:
I think we should see the bottom platform, so maybe move it "one platform height" up.